src/settings/MappingProfiles/MappingProfilesTransformationsModal/MappingProfilesTransformationsModal.js

```diff
@@ -119,19 +124,42 @@ export const MappingProfilesTransformationsModal = ({
   })), []);
 
   const handleSelectChange = useCallback(
-    transformations => setSelectedTransformations(transformations),
-    []
+    transformations => {
+      if (isSubmitButtonDisabled) {
```

Comment:
I thought that we enable the button after the change in the inputs only, does it make sense to that on clicking on checkboxes?

Reply:
Well, yes, but my idea was, that user might want to add empty instance transformation, in that case, it would be logically to enable submit button.

Reply:
And also, it's disabled from the start, so checking checkbox should def enable it.